src/main/java/com/example/task7/SampleController.java

```diff
@@ -0,0 +1,37 @@
+package com.example.task7;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.util.Map;
+
+@RestController
+public class SampleController {
+
+    @GetMapping("/names")
+    public CreateForm createForm(@RequestParam(value = "name", defaultValue = "name") String name, @RequestParam(value = "birthday", defaultValue = "0000") String birthday) {
+        return new CreateForm(String.format(name), String.format(birthday));
+    }
+
+    @PostMapping("/names")
+    public ResponseEntity<String> create(@RequestBody @Validated CreateForm form) {
+        URI url = UriComponentsBuilder.fromUriString("http://localhost:8080").path("/names/id")
+                .build()
+                .toUri();
+        return ResponseEntity.created(url).body("name successfully created");
```

Comment:
return ResponseEntity.created(url).body("name successfully created");
bodyの"name successfully created"を
```json
{
  "message": "name successfully created"
}
```
のようなjson形式で返却するように修正してみてください！ 🚀 
```suggestion
        return ResponseEntity.created(url).body("name successfully created");
```

Reply:
1734a81cf8954b7bbe280d24c0c9311e0d97f616
PATCHやDELETEと同じようにしてjson形式で返却するように修正しました。

Reply:
ばっちり！